recipes/r-mrmlm/meta.yaml

```diff
@@ -33,7 +33,7 @@ requirements:
     - cross-r-base {{ r_base }}    # [build_platform != target_platform]
   host:
     - r-base >=4.1
-    - r-bedmatrix
+    - r::r-bedmatrix
```

Comment:
No, we cannot do this (Conda Forge is standalone). Instead, this recipe will need to wait for #25307 to merge and start generating builds.